src/cs50/flask.py

```diff
@@ -14,20 +14,21 @@ def _wrap_flask(f):
 
     f.logging.default_handler.formatter.formatException = lambda exc_info: _formatException(*exc_info)
 
-    if os.getenv("CS50_IDE_TYPE") == "online":
+    if os.getenv("CS50_IDE_TYPE"):
```

Comment:
@kzidane is this change okay so that `TEMPLATES_AUTO_RELOAD` is enabled both online and offline, but `ProxyFix` is online only?

Reply:
LGTM!